testsupport/space/space.go

```diff
@@ -26,6 +26,13 @@ func CreateSpace(t *testing.T, awaitilities wait.Awaitilities, opts ...testspace
 // CreateSpaceWithRole initializes a new Space object using the NewSpace function, and then creates it in the cluster
 // It also automatically provisions MasterUserRecord and creates SpaceBinding for it
 func CreateSpaceWithRole(t *testing.T, awaitilities wait.Awaitilities, role string, opts ...testspace.Option) (*toolchainv1alpha1.Space, *toolchainv1alpha1.UserSignup, *toolchainv1alpha1.SpaceBinding) {
+	sp, sr, sb := CreateSpaceWithRoleSignupResult(t, awaitilities, role, opts...)
+	return sp, sr.UserSignup, sb
+}
+
+// CreateSpaceWithRole initializes a new Space object using the NewSpace function, and then creates it in the cluster
+// It also automatically provisions MasterUserRecord and creates SpaceBinding for it
+func CreateSpaceWithRoleSignupResult(t *testing.T, awaitilities wait.Awaitilities, role string, opts ...testspace.Option) (*toolchainv1alpha1.Space, *testsupport.SignupResult, *toolchainv1alpha1.SpaceBinding) {
```

Comment:
maybe we can rename it to something like `createSignupWithRole` ? And also update the comment.

Reply:
I forgot to update the comment, I'll fix that.

This method needs to be exported for being used in`parallel` and it is also creating the space, so the name you suggested won't work. But -I agree- we can find a better name.

Reply:
Sorry why does it needs to be exported ? I see the `CreateSpace` is used in the `parallel` test suite, this one is just referenced in this file. But I'm sure I'm missing something. Also it's a minor thing. 

Yeah naming is hard, I don't have a better name. Maybe `CreateSpaceSignupAndBindingWithRole` , or just keep the current name.

Reply:
Oh never mind . I'm blind . I saw now you are calling it from the test 🤦‍♂️ . Sorry for the confusion.

Reply:
But yeah maybe we can find a different name for the function.

Reply:
fixed comment here 322e7688618c5ca7f161a920a9a2242d7d9f5fda